master/reference/cni-plugin/configuration.md

```diff
@@ -180,3 +180,55 @@ When using the CNI `host-local` IPAM plugin, a special value `usePodCidr` is all
 ```
 
 When making use of the `usePodCidr` option, the Calico CNI plugin requires read-only Kubernetes API access to the `Nodes` resource.
+
+##### IPAM Manipulation with Kubernetes Annotations
+
+###### Specifying IP Pools
```

Comment:
I'd recommend shortening this section a bit to be more focused.  Here's an example:

```
##### Specifying IP Pools on a per-Pod basis

In addition to specifying IP Pools in the CNI config as discussed above, Calico IPAM supports specifying IP Pools per-Pod using the following Kubernetes annotations.

- `cni.projectcalico.org/ipv4pools`: A list of configured IPv4 Pools from which to choose an address.
- `cni.projectcalico.org/ipv6pools`: A list of configured IPv6 Pools from which to choose an address.

If provided, these IP Pools will override any IP Pools specified in the CNI config.

Note that any specified IP Pools must already have been created.

Example annotation:

...
```